hpy/devel/include/hpy.h

```diff
@@ -28,42 +28,42 @@
 
 /* ~~~~~~~~~~~~~~~~ HPyAPI declaration ~~~~~~~~~~~~~~~~ */
 
-/* We have three different kind of API functions:
-
-   HPyAPI_FUNC
-
-       Public API functions which are exposed to the user, e.g. HPy_Add or
-       HPyType_FromSpec. Generally speaking they are thiny shims dispatching
-       to the actual implementation:
-
-           - In CPython-ABI mode they directly call the corresponding Py* or
-             HPyAPI_IMPL equivalent, e.g. PyObject_Add or ctx_Type_FromSpec.
-
-           - In Universal-ABI mode, they always resolve to an indirect call
-             through HPyContext*, i.e. ctx->ctx_Add(...), which on CPython
-             dispaches to ctx_Add
-
-   HPyAPI_IMPL
-
-       CPython implementation for HPyAPI_FUNC functions. Generally speaking,
-       they are put in ctx_*.c files and they are prefixed by ctx_.
-
-       Some of these functions are needed by the CPython ABI mode, and by
-       CPython's implementation of hpy.universal: these can be found in
-       hpy/devel/src/runtime/ctx_*.c, e.g. ctx_Type_FromSpec and
-       ctx_Tuple_FromArray.
-
-       Some other are used ONLY by hpy.universal and can be found in
-       hpy/universal/src/ctx_*.c.
-
-    HPyAPI_HELPER
+/* We have three different kind of API functions: */
+
+/**
+ * Public API functions which are exposed to the user, e.g.
+ * HPy_Add or HPyType_FromSpec. Generally speaking they are thin shims
+ * dispatching to the actual implementation:
```

Comment:
does `sphinx_c_autodoc` support any kind of markup? E.g. it would be nice to render things like `HPy_Add` in fixed-width font

Reply:
We should be able to use any markup that Sphinx supports.

Reply:
Bullet lists added. In addition to just writing nicer markup, we also had to pin `docutils==0.16` to work around a docutils bug.